book/website/index.md

```diff
@@ -3,7 +3,7 @@
 
 *Welcome to The Turing Way handbook to reproducible, ethical and collaborative data science.*
 
-_The Turing Way_ project is providing its source files, open collaboration, and community-driven.
+_The Turing Way_ is an open science, open collaboration, and community-driven project.
```

Comment:
Hi Simon, I am not convinced that The Turing Way welcome page needs to be edited the way proposed here. We have agreed that Turing Way is not software, but it is an open project. I will add a preferred edit with "open science" (instead of "source", which seems to be the hook here) and merge.